### ISSUE TITLE
Feature: Add "python3" source type

### ISSUE THREAD

**Issue by jhermann** [jordansissel/fpm#1361]
Right now, you have to set these options for Python3 to make things work for 2017's *default* Python:

    --python-bin python3 --python-package-name-prefix python3

It'd be nice to have ``python3`` that behaves exactly like ``python``, but changes the defaults for the above options.

**Comment by shevegen** [comment]
Agreed. Perhaps have a compatibility python2 switch too and default to python3 since the latter seems to be finally more in use by now.

**Comment by jhermann** [comment]
Adding python2 for symmetry is good, but do not change "python" – no need to break existing stuff, and also all distros I know map python to python2, which is a good thing (again, no breakage).

**Comment by jordansissel** [comment]
I like this idea kinda... but what if python 3 is only available as
/usr/bin/python ? Thoughts?

**Comment by jhermann** [comment]
Then you are in a very exotic place and need to use the explicit options.

**Comment by mhjacks** [comment]
@jordansissel https://www.python.org/dev/peps/pep-0394/ specifies that /usr/bin/python is currently expected to be python2.

**Comment by jordansissel** [comment]
Ahh that PEP is informative. Thank you!

I agree with this proposal.

**Comment by jhermann** [comment]
Python2 is now officially dead for >2 months.
